### DISCUSSION TITLE
next_fast_len

### DISCUSSION THREAD

**Discussion by jecampagne** [jax-ml/jax#15200] [Q&A]
Hi, 
I do not see any "scipy.fft.next_fast_len" equivalent, I miss it? any plan to implement it?
Thanks

**Comment by jakevdp** [comment]
Hi - in general functions in `jax.scipy` are implemented on an as-needed basis. In this case, I believe the `next_fast_len` function is an implementation-dependent thing, so to implement it correctly in JAX would involve looking into the fft algorithm used by various backends (CPU, GPU, TPU). It might take a bit of digging.

**Reply by jecampagne** [reply]
Ok.
